extensions/parquet/base/src/main/java/io/deephaven/parquet/base/ParquetFileReader.java

```diff
@@ -49,7 +50,7 @@ public ParquetFileReader(final URI parquetFileURI, final SeekableChannelsProvide
         this.channelsProvider = channelsProvider;
         if (!parquetFileURI.getRawPath().endsWith(".parquet") && FILE_URI_SCHEME.equals(parquetFileURI.getScheme())) {
             // Construct a new file URI for the parent directory
-            rootURI = Path.of(parquetFileURI).getParent().toUri();
+            rootURI = convertFileToURI(new File(parquetFileURI).getParentFile());
```

Comment:
For example, at this callsite we _know_ file is a directory, and could call with `isDirectory=true`.